### ISSUE TITLE
L1 Capsule conversion

### ISSUE THREAD

**Issue by rnyoakum** [raceintospace/raceintospace#392]
According to https://github.com/raceintospace/raceintospace/issues/216#issuecomment-531840374

> During development of BARIS, there were plans to have a Zond capsule. If you had developed Voskhod and Soyuz, for 15MB you could convert the Voskhod to the Zond capsule, and it was a two-man vehicle. Its Safety matched Soyuz's safety, from what I've heard. There are still scattered references to it in the code, and we even have pictures for the Zond capsule (liftoff.abz.SVC7.png and rdfull.but.55.png) and mission patches for it (100-109). If we did implement something like this, I would probably want to rename it to L1 for historical accuracy.

I've seen the scattered references to this in the code, and it would be cool to add a planned yet unimplemented feature. Plus, giving the Soviets something unique to distinguish them from their staid American counterparts would serve as an inducement to get more players to embrace Communism.

Unfortunately, I don't know much about the original plans for the Zond/L1 capsule because it wasn't implemented. Consequently, I don't know what resources are available / what is missing.

I'd love to know
- What art resources currently exist for the Zond/L1?
- How would it have differed, in game mechanics, from the Voskhod?

**Comment by peyre** [comment]
I see you quoted from my guide!  That's most of what I know of the old plans for Zond.  As I understand it, a certain length of time after you started Soyuz (after having started Voskhod before), you were given the option to upgrade the Voskhod (that's i003 in help.cdr).  If you said yes, it cost 15MB and the Zond actually replaced Voskhod - so I guess the building would become the ZOND PROGRAM building.  I think the Zond had the same Max R&D and Safety as Soyuz.  This is what Erik Anderson wrote me back in 2005:

Another exception that didn't make the cut was the Zond.  The Zond is of
course a derivative of the Soyuz, reducing weight by taking off the orbital
module.  Actual implementation was a mess, though; 18 months (?) after you
bought the Soyuz, you'd get a dialog box that would ask you if you wanted to
pursue the Zond.  If so, you paid, got a safety factor equivalent to the
Soyuz, and it REPLACED the Voshkod.  So you still had to make up astronaut
groups for it, finish the R&D, and so forth.  You can see the problems this
would create for the code.  It was sorta working in a demo version, but was
pulled shortly thereafter.  Caused too many problems.

I've never seriously considered reimplementing the Zond, but if you feel like doing it, I think it would enrich the game.  It's an extra choice, after all--no one ever has to use it.  I do think we should call it the LOK, or maybe L1.

There are several pieces of artwork for Zond that I know about.  For R&D and Purchasing there is a drawing-board image for the capsule, **liftoff.abz.SVC7.png**, and the prototyped capsule, **rdfull.but.55.png**.  We also have Zond mission patches: 100-109.  I don't think there are any videos of the Zond; possibly it was dropped before they created the mission videos.  I suppose we could use Voskhod videos for it.

**Comment by peyre** [comment]
So, if we go through with this sometime, we'd have to work out the stats for the LOK: the Max R&D, MaxSafety, Unit Cost, and Unit Weight.  If we get serious about it, I could write to Erik and maybe Michael and see if they remember how it was supposed to be handled.  I assume from what Erik said above that when you start the capsule its safety is equal to the current Safety of your Soyuz, but it's independent afterward.  I expect that means it should have a Max Safety of 94% like Soyuz, but what should the Max R&D be?  Same as Soyuz, or a bit less?  My guess is that they intended it to be equal to Soyuz, but intuitively I would expect it to be a little lower for game balance, maybe 83% like Gemini.

Re. the unit weight: According to [url=https://spaceflight101.com/spacecraft/soyuz-tma-m/]spaceflight.com[/url], the Soyuz TMA weighs a total of 7150 kg (15,763 lbs), divided into:
1300 kg (2866 lbs) - Orbital Module
2950 kg (6503 lbs) - Entry Module
2900 kg (6393 lbs) - Instrumentation Module

Now the L1/LOK capsule was a Soyuz minus the Orbital Module plus some extra gear for docking, etc.  The extra could be abstracted off as part of the DM.  Removing the orbital module leaves us at a capsule weight of 5850 kg (12,897 lbs).  Now I see the Soyuz shows as 1900 lbs in the game(!), so we're clearly not going off actual figures.  For RIS, we could base it off that.  The orbital module is about 18% of the Soyuz's weight, so it would come to about 1550 lbs, about 150 lbs heavier than the Voskhod.  That sounds about right to me, but it would take some figuring to see what that would do to the rocketry needed for various missions.

Also, of course, an LOK could not be used for a Soyuz Lunar Landing.

**Comment by peyre** [comment]
Another question to consider is whether a Zond capsule would need a kicker to leave Earth orbit.  I'm inclined to think so, for game balance purposes.

It looks like a 1550lb capsule would still need the N1 rocket for the Historical Lunar Landing, so that might be a good weight for it.

**Comment by peyre** [comment]
Erik has kindly clarified what he remembers of it:

Since it wasn’t fully implemented, it also wasn’t balanced.  And we didn’t do any work with the board game to balance it either, so anything is going to have to be a guess that has to be modified in light of play testing.

At some point, you get the option of paying your money, and this deletes the Voskhod program and Zond takes its place.

Yes, safety factor is set as equal to the Soyuz, or perhaps minus a bit.  They’re independent programs thereafter.  Max R&D probably 2% less than Soyuz.  Max duration definitely less than Soyuz, so E.  (Frankly, Voskhod ought to be less still, but let’s not go there).
Unit cost 9 or 10, yes.  Probably 9.

Weight is whatever exactly fills up a Proton for a Manned Lunar Pass.  (But with no boosters?  With no kicker?  I think that’s right...which would make it 1500, if I remember correctly).

There was discussion on whether it needed a kicker or not.  Since it’s the same propulsion system as a Soyuz, it’s hard to pretend it does.  So I think that’s the way we went...but I’m not certain.  Making a two-person capsule that doesn’t need a kicker creates obvious coding issues.
   Requiring a kicker creates a weight problem...if we take a Proton and Manned Lunar Pass as the sizing constraint for Zond (which it was, in the real world), then Zond plus a Kicker-A (if needed) needs to come out to 1500.  So if it needs a kicker, Zond only weighs 1200...too light.  Or 2200 (Proton with boosters), too heavy.  So...probably 1500 weight and no kicker.  It WOULD need a docking module, for any mission that requires docking.  Since it got rid of, you know, the orbital module.  Which is what a Soyuz uses to dock with.
   R&D cost per die...I have no idea.  Yes, 4 is a good guess.  Doesn’t matter much I suppose, since if you play it right you don’t research it much.

**Comment by peyre** [comment]
Come to think of it, if we do this we'll also have to come up with stats for the Basic Version.  Not saying we shouldn't do this; it's just something we'll have to work out if we go ahead with this.

**Comment by peyre** [comment]
Having a closer look at what Erik said about capsule weight, about it having to just fit on a Proton rocket: it looks like that's just how Voskhod is done, both in the Historical and Basic Models.  That is, the capule + Kicker-A just fits on a non-boosted Proton.  So we can't just do the same for Zond, I think: it really needs to have a higher weight than Voskhod.  A weight of 1500 (or higher, for that matter) would require a boosted Proton for lunar passes and orbitals, and would need an N1 for Earth orbital LM tests.  Historical Manned LL would require a boosted N1.  All that also goes for a figure of 1300 in the Basic Model (vs. Voskhod 1200 and Soyuz 1600), except the Historical LL would work on a plain N1.

**Comment by peyre** [comment]
Another thought about this idea: if you receive the offer to upgrade Voskhod to LOK/L1/Zond, there should probably be an option in Hardware Purchase to allow you to change your mind later and proceed with the upgrade.

**Comment by peyre** [comment]
Also, I003 in the help file would have to be updated, and I can do that easily enough.  I think I'd probably change it to something like:
ENGINEERING HAS COMPLETED DESIGNS FOR
A SMALLER, TWO-SEAT VERSION OF THE
SOYUZ CAPSULE, WHICH WOULD REPLACE
THE VOSKHOD PROGRAM AND COST 15 MB'S.
DO YOU WISH TO REPLACE YOUR VOSKHOD
PROGRAM WITH THE NEW LOK CAPSULE?

**Comment by peyre** [comment]
I thought of a related question recently: should Zond/L1/LOK be allowed to use the C-Kicker?  I'm inclined to think not, since it probably doesn't need that advantage.  Game reasoning could be that historically, that capsule was designed specifically for the N1-L3 program to be used with the LK lander.

**Comment by hweimer** [comment]
Well, N1-L3 is the historical lunar landing with the Soyuz capsule (Soyuz 7K-LOK in real life), right? Kicker C looks like the previous Soyuz A-B-C approach moved to LOR for balancing reasons (I think you mentioned this to me at some point). Since the L1 was never meant to be used for a landing, it's difficult to say, but again for balancing reasons, I'm also inclined to think it shouldn't be possible to use the Kicker C.

**Comment by hweimer** [comment]
True, but Gemini was less of a PR stunt, its goal was mostly to develop the capability to perform rendezvous and docking in space, later to be used in the Apollo lunar landings. Besides, there were also serious proposals for a Gemini lunar landing, which would have been both cheaper and at an earlier date. However, NASA's mindset was not so much "how can we get the job done the quickest way possible" (like in the game), but rather "what's the best stuff we can get funded by Congress while still meeting JFK's deadline".

**Comment by peyre** [comment]
Duncan makes a good point, that Gemini was patterned off Mercury, as Voskhod was from Vostok.

However as Hedrick points out, Gemini was a new spaceship _patterned on_ Mercury, whereas Voskhod was effectively a variant of Vostok.  The two that flew, as I understand it, were actually Vostok capsules that were modified: a retrorocket was bolted on, extra seats were added, and (for Voskhod 2) an airlock was added.  Whereas Gemini was a whole new spacecraft, much bigger and with added capabilities.  I think we'd be justified (if we want to) in calling Voskhod a Vostok variant and Gemini a different spacecraft.

**Comment by hweimer** [comment]
I think we will always have some inconsistencies simply because the U.S. and Soviet space programs were not exactly the same. However, in terms of their capabilities (except for the maximum duration), it seems to me that Gemini/L1 is a closer match than Gemini/Voskhod. So maybe it would make sense to make L1 the Soviet two-seater and have Voskhod as an optional upgrade to the Vostok program. Ignoring all coding horrors at this point, of course.

**Comment by peyre** [comment]
Yes, "coding horrors" is a good way to put it.  A lot of things in the game are done for simplicity of coding of course.  I like the idea of making Voskhod a variant/upgrade of Vostok, and having LOK/L1 replace the current Voskhod.  An upgraded Vostok would require a bunch of special exceptions though.

**Comment by hweimer** [comment]
Actually, it's not that bad. The mission handling of the 'nauts is essentially independent of the program being used.

![1 xwd](https://user-images.githubusercontent.com/68769393/131402600-527f22ec-9456-4a09-9aad-3bed8e9a617b.png)
![2 xwd](https://user-images.githubusercontent.com/68769393/131402603-56db3662-3db9-4750-ad4f-dd22f7ddf89d.png)
![3 xwd](https://user-images.githubusercontent.com/68769393/131402605-e0fe9a48-8768-4157-b7c6-0fbf2dababc8.png)
![4 xwd](https://user-images.githubusercontent.com/68769393/131402608-6eff508c-bfe3-41f5-9cea-a77c7351b6f3.png)

**Comment by peyre** [comment]
A roll of -2!  I've never seen that before.  I assume this mission faced a milestone and/or duration penalty.

So that wouldn't be as difficult to do as it seemed to me?  If it's practical do implement it, that might be a good plan.

**Comment by peyre** [comment]
I suppose we could do something like this in R&D.  Maybe also in the Vostok Program screen.

![VostokUpgrade](https://user-images.githubusercontent.com/3118014/131537643-a8855584-4abc-43aa-90f7-a9d094365004.png)
```
draw_string(275, 144, "UPGRADE");
IOBox(272, 136, 317, 149);
```

**Comment by peyre** [comment]
Hey, just found an artifact in the code that's sort of relevant.  According to port.cpp (775-6), the Zond conversion was supposed to become an option 6 turns after you prototyped Soyuz.

Not that we would be bound to follow that, but it's good to know.

**Comment by peyre** [comment]
Here's a little something for the Capsule Program screen, if we should ever implement this idea.  In place of
![aprog 1 3](https://user-images.githubusercontent.com/3118014/155850782-b9ac6ba3-3318-414b-b80b-587e66c98cdb.png)

for Soyuz, we could have this for LOK:
![aprog 1 31](https://user-images.githubusercontent.com/3118014/155850809-b3ca0bb2-bc65-463d-8315-014e389e7965.png)

**Comment by peyre** [comment]
Incidentally, if we should sometime implement Voskhod being an incremental upgrade of Vostok, should the obsolete rule still apply to it?  (That is, if you have a more advanced program, anyone who's been in Mercury/Vostok for over 4 turns starts to lose morale as they feel they're stuck in an obsolete program.)  I'm thinking it probably should, since it's still a Vostok capsule with just a few extra capabilities, still very limited.

**Comment by peyre** [comment]
Got to thinking about it this morning.  You know, that image above for the Zond looks kind of dopey.  Maybe we should use something more closely based on Soyuz?  Like this maybe.

![rdfull but 50](https://github.com/raceintospace/raceintospace/assets/3118014/ada0bab2-f9c0-4dff-a770-9f38fb235d46)

**Comment by peyre** [comment]
Or, rather than my original suggestion of using Voskhod videos for the LOK, maybe we could mod the existing Soyuz ones, essentially chopping off the orbital module.

**Comment by peyre** [comment]
In light of my learning today that there's a Zond Program building tucked away in the code, we'd have to figure out how to go about all this.  If Voskhod becomes a variant of Vostok, maybe we stop using the Voskhod building and use the Zone one for the LOK/L1 Program.

**Comment by peyre** [comment]
Incidentally, one of the shortcuts Korolyov had to take to get the Voskhod into orbit was to leave off the Launch Escape System, so if something went badly wrong with the rocket, there was no way to eject the capsule to an emergency landing (much like the Space Shuttle later on).  I don't know if we would want to model that in the game or not.
